components/Primitives/samples/DockPanelSample.xaml

```diff
@@ -36,7 +36,9 @@
         <controls:DockPanel x:Name="SampleDockPanel"
                             Grid.Column="1"
                             Background="{ThemeResource CardBackgroundFillColorSecondaryBrush}"
-                            LastChildFill="False">
+                            HorizontalSpacing="5"
+                            LastChildFill="False"
+                            VerticalSpacing="5">
```

Comment:
@Arlodotexe did we want to expose these as sliders/bound properties? (forget if we have sliders yet) Otherwise, maybe use 4 or 8 for better scaling?

Reply:
I thought about it, but this is one of the samples where the controls are already in the Sample View rather than in an Options Pane. The entire things needs to be moved over to a custom options pane.